DirectEmployers/MyJobs
# [PD-3453] cleanup header component code

## What's this PR do?
PR cleans up the Header Component Code

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the analytics app or view the code changes

## How should this be manually tested?
Pull down the repository and make sure that they code is still working. You can also view the source code and see the code changes

## Any background context you want to provide?
